java/piranha/src/test/java/com/uber/piranha/CorePiranhaTest.java

```diff
@@ -499,4 +501,201 @@ public void testIgnoresPrefixMatchFlag() throws IOException {
             "class TestClassPartialMatch { }")
         .doTest();
   }
+
+  /**
+   * [https://github.com/uber/piranha/issues/141]
```

Comment:
Again, I'd move the issue links to the end, start with a line on what is being tested (which can be the existing next line)